src/main/java/com/example/bank/bankTransfer/transfer/TransferService.java

```diff
@@ -19,71 +22,96 @@
 @Getter
 @Slf4j
 public class TransferService {
+    private final TransferHistoryService transferHistoryService;
+    private final TransferHistoryRepository transferHistoryRepository;
     private final AccountRepository accountRepository;
     private final RateClient rateClient;
 
     public void bankTransfer(TransferRequest transferRequest) {
         TransferValidationUtils transferValidationUtils = new TransferValidationUtils();
-        @NonNull Long fromAccountId = transferRequest.fromAccountId();
-        @NonNull Long toAccountId = transferRequest.toAccountId();
+        @NonNull Long senderAccountId = transferRequest.senderAccountId();
+        @NonNull Long receiverAccountId = transferRequest.receiverAccountId();
```

Comment:
wyobraz sobie ze masz aplikacje bankowa i jej frontend.
Masz pole do wpisania ID konta czy numeru konta ? 
Nie mozesz mapowac po ID bo front nie bedzie wiedzial o ID jezeli jest to transfer miedzy bankowy. 
Wszystko powinno sie odbywac po numerze konta nie po ID konta.
Nawet jezeli jest to wewnetrzny przelew to i tak powinienes mapowac po  numerze. Bo uzytkownik numer konta wprowadzi ale skad ID bedzie znal? 
Musisz myslec tak jakby to API mialo byc wykorzystywane przez jakis frontend pozniej.